README.md

```diff
@@ -305,6 +305,24 @@ Run the tool through the context menu on a file or directory:
 
 </details>
 
+## Troubleshooting
+
+### “spawn jpegoptim ENOENT”, “spawn guetzli ENOENT”, etc
+
+Make sure that the [ignore-scripts](https://docs.npmjs.com/cli/v6/using-npm/config#ignore-scripts) option is not active.
+
+More info: <https://github.com/funbox/optimizt/issues/9>.
```

Comment:
You probably may replace it with `#9`, or `[#9](https://github.com/funbox/optimizt/issues/9)`. It will look better.

Reply:
Yeah, but then i cannot open this link from the IDE

Reply:
🤔 

![image](https://user-images.githubusercontent.com/6537798/107950992-3fc11100-6fa0-11eb-88bc-8435b2f1020c.png)

Ctrl+Click on the link (not on the title of the link) works for me. Does it work on macOS with Cmd or something?